_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -0,0 +1,549 @@
+the link:https://github.com/gruntwork-io/cloud-nuke[`cloud-nuke
+defaults-aws`] command to remove all default VPCs from all regions.
+
+[[configure_monitoring]]
+=== Configure monitoring
+The Monitoring section of the Benchmark centers on a collection of
+link:https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/MonitoringLogData.html[CloudWatch Logs Metric
+Filters]. Gruntwork has simplified this section to a single module: the
+link:https://github.com/gruntwork-io/cis-compliance-aws/blob/master/modules/cloudwatch-logs-metric-filters/README.adoc[`cloudwatch-logs-metric
+-filters` wrapper module]. It will create and configure all the CloudWatch Logs metric filters necessary for
+compliance with the Benchmark.
+
+[[configure_networking]]
+=== Configure networking
+The networking section involves a paltry four recommendations. We don't consider this section to be sufficient
+to ensure a secure networking configuration. For a deeper dive, refer to Gruntwork's
+link:https://gruntwork.io/guides/networking/how-to-deploy-production-grade-vpc-aws/[How to deploy a
+production-grade VPC on AWS] guide, which includes recommendations for segmentation using network ACLs,
+security groups, and remote access. Moreover, our link:https://gruntwork.io/reference-architecture/[Reference
+Architecture] can get you up and running with a secure network configuration immediately.
+
+If you're using Gruntwork's link:https://github.com/gruntwork-io/module-vpc[VPC module] for your VPCs, three
+of the four recommendations in this section are already taken care of! By default, none of our modules allow
+access to ports 22 or 3389 from the world, and our architecture has a least-privileges-based routing
+configuration by default.
+
+The only necessary step here is to run the link:https://github.com/gruntwork-io/cloud-nuke[`cloud-nuke
+defaults-aws`] command to remove all default security groups from all VPCs in all regions.
+
+[[next_steps]]
+== Next steps
+
+Congratulations! If you've made it this far, you should have achieved compliance with the CIS AWS Foundations
+Benchmark. Now it's time to confirm that your configurations are correct and you didn't miss any steps. You
+can enable the link:https://aws.amazon.com/security-hub/[AWS Security Hub] to check your account
+link:https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-standards.html[for compliance with
+the AWS CIS Foundations Benchmark]. The Security Hub runs the exact audit steps specified in the Benchmark
+using AWS Config managed rules. By enabling the Security Hub, you can track your compliance efforts and be
+notified if any recommendations have not been implemented. Follow the AWS documentation to enable Security
+Hub.
+
+
+[[traceability_matrix]]
+== Traceability matrix
+Use the table below as a quick reference to map the CIS AWS Foundations Benchmark recommendations to the
+sections above.
+
+
+[cols="^1,<10,15",format=csv]
+|===
```

Comment:
I was about to comment that we should be clear about which recommendation each action maps to, and then saw this chart. Very nice addition!